src/components/page/MembersPage.tsx

```diff
@@ -17,9 +18,7 @@ export const MembersPage = async ({ id }: IMembersPageProps) => {
           <Button className="flex justify-center items-center w-12 h-12">
             <Icon icon={MenuMeatballIcon} size={5} />
           </Button>
-          <Button className="flex justify-center items-center w-12 h-12">
-            <Icon icon={PlusIcon} size={4.5} />
-          </Button>
+          <InviteMemberButton></InviteMemberButton>
```

Comment:
Since button doesn't have children, 
I suppose we can just self-close it like this 🔽 !
`<InviteMemberButton/>`

Reply:
@kanta1207 
Thank you for the advice! I fixed it!